packages/suite/src/actions/settings/walletSettingsActions.ts

```diff
@@ -83,3 +87,47 @@ export const removeBlockbookUrl = (payload: BlockbookUrl) => ({
     type: WALLET_SETTINGS.REMOVE_BLOCKBOOK_URL,
     payload,
 });
+
+export const setTorBlockbookUrls = () => async (dispatch: Dispatch, getState: GetState) => {
+    const { blockbookUrls } = getState().wallet.settings;
+    const noTouchyCoins = blockbookUrls.filter(u => !u.tor).map(u => u.coin);
+    const coins = NETWORKS.filter(n => !n.accountType && !noTouchyCoins.includes(n.symbol)).map(
+        n => n.symbol,
+    );
+
+    const torBlockbook: BlockbookUrl[] = [];
+    // Disabling this ESLint rule here because it doesn't make sense for this scenario.
+    // It suggests using forEach which not waiting for the loop to finish before going
+    // to the next statement, resulting in `torBlockbook` to be empty.
+    // eslint-disable-next-line no-restricted-syntax
+    for await (const coin of coins) {
+        try {
```

Comment:
connect should never throw an error (if this is the reason why this try/catch exists), 
it should return `success: false`

Reply:
Wasn't sure here, added the try/catch as a precaution. Good to know for the future, thanks!